daemon/daemon_test.go

```diff
@@ -38,3 +40,76 @@ func TestFailedReload(t *testing.T) {
 		t.Fatalf("Should have errored on changed listen port")
 	}
 }
+
+var localServerPort = ":8081"
+var localServerHost = "http://localhost" + localServerPort
+var localProxyHost = "http://localhost:6634"
+
+func setUpDaemonWithLocalServer() error {
+	// Set up a local server that 404s everywhere except one route.
```

Comment:
I might expand this comment to say ", in particular on `/sphinx/health/check`." So that it is clearer how your test works.